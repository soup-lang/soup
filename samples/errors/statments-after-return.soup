func write(x)
    if x = 10
        return
        write(x)
    console.log(x = 10)

write(10)
