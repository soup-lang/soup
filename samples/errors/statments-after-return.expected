samples/errors/statments-after-return.soup:4
    block already terminated at Line 3
